test/unit/account_test.rb

```diff
@@ -746,6 +746,16 @@ def test_smart_destroy_master
     service1.update(state: Service::DELETE_STATE)
     assert_same_elements [c2], account.accessible_proxy_configs.current_versions
   end
+
+  test "allow underscore in org_name" do
+    account = FactoryBot.build(:simple_account, org_name: 'name@._+1')
+    assert account.valid?
+  end
+
+  test "there should be at least one alphanum character anywhere in the org_name" do
+    account = FactoryBot.build(:simple_account, org_name: '.?+@')
+    assert_not account.valid?
```

Comment:
I would also add this check here:
```
assert_includes account.errors.messages[:org_name], "should contain at least one alphanumeric character"
```

Reply:
Okay if we are adding there we can add here to so if it fails it will fail together

Reply:
Done changes so resolving